.travis.yml

```diff
@@ -9,8 +9,11 @@ julia:
   - nightly
 notifications:
   email: false
-script:
-  - if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
-  - julia -e 'Pkg.clone(pwd()); Pkg.build("DualNumbers"); Pkg.test("DualNumbers"; coverage=true)'
+# Workaround for https://github.com/travis-ci/travis-ci/issues/4942
+git:
+  depth: 99999
+#script:
+#  - if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
+#  - julia -e 'Pkg.clone(pwd()); Pkg.build("DualNumbers"); Pkg.test("DualNumbers"; coverage=true)'
```

Comment:
As I understand it, the preferred option is to have this portion commented; it's in [the Example repo](https://github.com/JuliaLang/Example.jl/blob/master/.travis.yml#L11) and it's [generated by PkgDev](https://github.com/JuliaLang/PkgDev.jl/blob/4fcf2f5f45364f5bbfd9b3dd644f17f405aa259d/src/generate.jl#L233). The advantage is that if ever the default changes, it's best not to override it, because a change to the default might be reflective of a change in Travis itself that wouldn't be caught with this script.

The workaround for the Travis depth bug should work in either case.